README.md

```diff
@@ -103,11 +103,17 @@ Remember to enable `deflate_module` and `filter_module` modules in your Apache
 configuration file.
 
 ### Migrating from MySQL to SQLite
+ ,
+Compared to SQLite, MySQL is relatively complicated to administer, brings in many unneeded dependencies, and consumes more resources.  Additonally, as a network service, poorly configured installations have the potential
```

Comment:
You could argue with the use of "many", possibly, which does not soften the comparison like much of the rest of the sentence does, with "compared to", and "relatively", etc.